### PR TITLE
Hmj fastsent

### PR DESCRIPTION
**Pull request by 544895340**
Removed duplicate code in FastSent

### CODE REVIEW

**Comment by piskvorky** [comment]
Thanks; what's "Hmj"? I'm not familiar with that acronym.

Also, description says "removed duplicate code", but diff shows new code being added. What is this PR about?

**Comment by 544895340** [comment]
@piskvorky HMJ is my name. I removed duplicate code in the second last commit (4a4bffd), you may refer to that. The last commit is because I forgot to remove the comment of the original main function. I used a simple test to quickly ensure that nothing is wrong after my modification.

File: gensim/models/fastsent.py [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+# Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""
+FastSent model for sentence representation.
+
+The FastSent algorithm was originally inspired by the C package https://code.google.com/p/fastsent/.
+
+For a blog tutorial on gensim fastsent, which is closely related, visit http://radimrehurek.com/2014/02/fastsent-tutorial/
```
Comment (tmylk) [inline gensim/models/fastsent.py]: non-existent links

File: gensim/models/fastsent.py [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""
+FastSent model for sentence representation.
+
+The FastSent algorithm was originally inspired by the C package https://code.google.com/p/fastsent/.
```
Comment (tmylk) [inline gensim/models/fastsent.py]: non-existent links

**Comment by tmylk** [comment]
@544895340 Please add changelog.md and commit the 300-line version of refactored FastSent

**Comment by 544895340** [comment]
I tried to refactor the FastSent, but I have no time to do this temporarily. If someone is interested, you can do the following work:
1. Remove dupicate code in Cython Files.
2. Refactor the initialization of FastSent.

**Pull request closed without merging**